node/src/node_configurator/configurator.rs

```diff
@@ -863,25 +865,39 @@ mod tests {
         )));
         let (recorder, _, _) = make_recorder();
         let recorder_addr = recorder.start();
-        let mut subject = Configurator::new(data_dir, false);
-        subject.node_to_ui_sub = Some(recorder_addr.recipient());
-        subject.new_password_subs = Some(vec![]);
 
-        let _ = subject.handle_change_password(
+        let mut configurator = Configurator::new(data_dir, false);
```

Comment:
You should change the `configurator` to `subject`. It's just the convention. Thanks for fixing the issues with the blank lines.